src/Avalonia.Controls/WindowTransparencyLevel.cs

```diff
@@ -33,9 +33,14 @@ private WindowTransparencyLevel(string value)
     public static WindowTransparencyLevel AcrylicBlur { get; } = new(nameof(AcrylicBlur));
 
     /// <summary>
-    /// The window background is based on desktop wallpaper tint with a blur. This will only work on Windows 11 
+    /// The window background is based on desktop wallpaper tint with a light blur. This will only work on Windows 11 
     /// </summary>
-    public static WindowTransparencyLevel Mica { get; } = new(nameof(Mica));
```

Comment:
Removing this value is a breaking change and can't be accepted mid 11.x.

Would be reasonable to keep singe "Mica", but change it behavior depending on the window theme variant? 
See https://github.com/AvaloniaUI/Avalonia/blob/432dafac91dc4cb61ffd338f7d8afcf5d4ab6540/src/Windows/Avalonia.Win32/WindowImpl.cs#L791

Reply:
Mica should automatically switch based on theme variant. I don't think there should be two entries for this and there aren't in WinUI either I believe.

Reply:
Changed it, wrote it before ThemeVariants were available :D